core/src/main/scala/kafka/raft/KafkaMetadataLog.scala

```diff
@@ -358,6 +485,7 @@ final class KafkaMetadataLog private (
     expiredSnapshots: mutable.TreeMap[OffsetAndEpoch, Option[FileRawSnapshotReader]]
   ): Unit = {
     expiredSnapshots.foreach { case (snapshotId, _) =>
+      info(s"Marking snapshot $snapshotId for deletion")
```

Comment:
Let's either include the topic partition or the log dir of this log.

Reply:
I assumed the class included some common log info like that, but looks like it doesn't. I'll add it here and open a separate PR to add some basic fields to the logIdent.

Reply:
I went ahead and added a logIdent to this class